### PR TITLE
7.0 pep8 fix

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Still some PEP8 erros:

```
./sale_dropshipping/purchase.py:98:73: E901 SyntaxError: invalid syntax
./sale_quotation_number/sale_quotation.py:22:1: F401 'fields' imported but unused
./sale_cancel_reason/model/sale.py:23:1: F401 '_' imported but unused
./sale_condition_text/condition.py:8:1: F401 'pooler' imported but unused
./sale_condition_text/sale_order.py:40:21: F821 undefined name '_'
./sale_condition_text/sale_order.py:41:21: F821 undefined name '_' 
```

Regards.

**Comment by eLBati** [comment]
On 08/07/2014 01:29 PM, Pedro M. Baeza wrote:

> Still some PEP8 erros:

Fixed.
Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +20,7 @@ install:
 
 script:
   - travis_run_flake8
-  - travis_run_tests
+  - travis_run_tests $HOME/product-attribute
```
Comment (bwrsandman) [inline .travis.yml]: `$HOME/product-attribute` should no longer be needed

**Comment by bwrsandman** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (eLBati) [inline .travis.yml]: > |$HOME/product-attribute| should no longer me needed

Trying without

**Comment by eLBati** [comment]
It works without `$HOME/product-attribute` :clap:

But we ran into https://github.com/OCA/maintainer-quality-tools/issues/60 :pensive:

**Comment by pedrobaeza** [comment]
As the target of this PR is PEP8, I give my :+1:

**Comment by rvalyi** [comment]
:+1: thanks!

**Pull request merged**